django_mailbox/utils.py

```diff
@@ -64,7 +64,7 @@ def get_settings():
         ),
         'default_charset': getattr(
             settings,
-            'DJANGO_MAILBOX_default_charset',
+            'DJANGO_MAILBOX_DEFAULT_CHARSET',
             'iso8859-1',
         )
```

Comment:
Django settings case is free, and uppercase is just a convention...
We need to keep supporting the malformed setting, looking for both cases:
```
        'default_charset': getattr(
            settings,
            'DJANGO_MAILBOX_DEFAULT_CHARSET',
            # Keep support for initially malformed setting name
            getattr(
                settings,
                'DJANGO_MAILBOX_default_charset',
                'iso8859-1',
            )
        )
```

Reply:
After investigation, it seems that Django is filtering settings that are not uppercase.
It would mean we don't need to handle the transition (like I tried in #280).

Can someone confirm that?

Reply:
https://docs.djangoproject.com/en/dev/topics/settings/#creating-your-own-settings